yarn-project/l1-artifacts/scripts/generate-artifacts.sh

```diff
@@ -32,6 +32,13 @@ CONTRACTS=(
   "l1-contracts:ExtRollupLib"
 )
 
+# Read the error ABI's once and store it in COMBINED_ERRORS variable
+COMBINED_ERRORS=$(jq -s '
+    .[0].abi + .[1].abi |
+    unique_by({type: .type, name: .name})
+' \
+    ../../l1-contracts/out/Errors.sol/Errors.json \
```

Comment:
maybe combine these while we are here?

Reply:
The split is currently with `core` and `governance` the paths just end up looking a bit funky in the foundry `out` dir.

Think we should keep the split for now.